casr/src/bin/casr-libfuzzer.rs

```diff
@@ -142,7 +144,19 @@ fn main() -> Result<()> {
             "casr-gdb"
         }
     };
-    let tool = util::get_path(tool)?;
+    let tool_path = util::get_path(tool)?;
+
+    if !gdb_args.is_empty() && tool != "casr-gdb" && tool != "casr-san" {
+        bail!("casr-gdb-args option is provided with incompatible tool (casr-python, casr-java or casr-js).");
```

Comment:
, or

Reply:
However, it's better to state compatible tools

Reply:
Specified compatible tools, but I am not sure i have mentioned all of them